lineage/data/Lineage_collections.py

```diff
@@ -185,16 +171,13 @@ def popout_single_lineages(lineages):
 taxb42 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00802_B4_1_V4.xlsx")]
 taxb422 = [LineageTree(list_of_cells, E) for list_of_cells in import_Heiser(path=r"lineage/data/heiser_data/new_version/AU00802_B4_2_V4.xlsx")]
 
-taxs = taxb40 + taxb402 + taxb41 + taxb412 + taxb42 + taxb422
-Tax2uM = popout_single_lineages(taxs)
```

Comment:
Should `Tax2uM` be assembled, just not calling `popout_single_lineages`?

Reply:
fixed.